packages/wrangler/src/user/user.ts

```diff
@@ -1071,7 +1078,7 @@ export async function login(
 	complianceConfig: ComplianceConfig,
 	props: LoginProps = {
 		browser: true,
-		callbackHost: "localhost",
+		callbackHost: "0.0.0.0",
```

Comment:
I am not convinced that this is an appropriate default.
This IP address has special meaning and can result in the server listening to requests on all hosts to the machine.

I think it should be opt-in rather than the default.